index.html

```diff
@@ -2102,55 +2017,26 @@ <h2>Disputes</h2>
 
         <p>
 The time may come when an <a>entity</a> wants to dispute a <a>credential</a>
-issued by the issuer. There are at least two different cases to consider:
-a subject disputes a claim made by the issuer, e.g. the address property is out
-of date, or an entity disputes a (false) claim made by the issuer about a
-different subject, e.g. an imposter is claiming the entity's social security
-number. Only the subject of a verifiable credential is entitled to issue a
-"DisputeCredential". A "DisputeCredential" issued by anyone other than the
-subject SHOULD be disregarded by the verifier, unless the verifier has some out
-of band means of ascertaining the truth of the dispute. This is to prevent
-denial of service attacks whereby an attacker falsely disputes a true claim.
-
-The mechanism for issuing a "DisputeCredential"
-is the same as issuing a regular credential except that the <a>subject</a>
-identifier for the <a>claim</a> in the "DisputeCredential" is the identifier of
-the disputed credential.  For example, if a credential with an identifier of
-<code>https://example.org/credentials/245</code>
-is disputed, an entity may issue one of the following credentials. In the
-former case, the subject might present this to the verifier along with the
-disputed credential. In the latter case, the entity might publish the
-"DisputeCredential" in a public venue to make it known that the credential
-is disputed.
-        </p>
-
-        <pre class="example nohighlight" title="Subject disputes a credential">
```

Comment:
Did you mean to remove all this? It seems like a reversion of the current spec and the PR that was pulled in to clean this DisputeCredential stuff up.

Reply:
No I did not mean to remove this. This was a mistake

Reply:
I have just checked my master copy and I did not remove this. It was already removed.
So I guess I did not edit the latest version.
It is probably better to simply remove my PR and I will start all over again.